Description: use larger RSA keys in tests
Bug-Debian: https://bugs.debian.org/912245
Author: Cédric Boutillier <boutil@debian.org>
Last-Update: 2019-01-02

--- a/spec/nio/selectables/ssl_socket_spec.rb
+++ b/spec/nio/selectables/ssl_socket_spec.rb
@@ -12,7 +12,7 @@
 
   let(:addr) { "127.0.0.1" }
 
-  let(:ssl_key) { OpenSSL::PKey::RSA.new(2048) }
+  let(:ssl_key) { OpenSSL::PKey::RSA.new(4096) }
 
   let(:ssl_cert) do
     name = OpenSSL::X509::Name.new([%w[CN 127.0.0.1]])
